src/App.tsx

```diff
@@ -1,7 +1,13 @@
 import React from "react";
+import { Container } from "react-bootstrap";
+import MainRouter from "./components/MainRouter/MainRouter";
 
 function App() {
-  return <div>Hello</div>;
+  return (
+    <Container>
```

Comment:
You can also add some `ErrorBoundary` here for catching different component errors